packages/discord-bot/src/utils/settingsUtil.ts

```diff
@@ -0,0 +1,91 @@
+import { Setting } from './api-schema';
+import { apiClient } from './api';
+
+export function settingValueRealized(
+  setting: Setting[]
+): string | string[] | boolean | number | number[] | undefined {
+  const { value, type } = setting[0];
+  if (!value) return undefined;
+
+  switch (type) {
+    case 'Integer':
+      return Number.parseInt(value);
+    case 'Float':
+      return Number.parseFloat(value);
+    case 'Boolean':
+      return value === 'true' ? true : false;
+    case 'IntegerList':
+      return value.split(',').map((v: string) => Number.parseInt(v.trim()));
+    case 'StringList':
+      return value.split(',').map((v: string) => v.trim()) as string[];
+    case 'Image':
+      return `${process.env.API_URL as string}:${
+        process.env.API_PORT as string
+      }/api/settings/uploads/${value}`;
+    case 'JSON':
+      return value ? JSON.parse(value) : [];
+    default:
+      return value;
+  }
+}
+
+export const getDefaultSetting = async (
```

Comment:
Is this needed? All settings already include default values in the DB. 

https://github.com/givepraise/praise/blob/7ad7e1ed6aac1cb55e3ad629843dceacd9154ae8/packages/api/src/settings/schemas/settings.schema.ts#L71

Reply:
I added this as a fallback in case the API doesn't return a valid setting due to some reason.